resources/views/components/dashboard/item/index.blade.php

```diff
@@ -0,0 +1,36 @@
+@props(['item' => [], 'key' => ''])
+<component {{ $attributes->merge([
+    'class' => 'flex flex-wrap items-center gap-x-6 rounded border bg-white px-8 py-4 text-left',
+    'href' => $item['href'] ?? '',
+    'is' => isset($item['href']) ? 'a' : 'button',
+]) }}>
+    @if ($item['icon'] ?? false)
+        <x-icon
+            class="h-6 stroke-[1.5px]"
+            name="{{ $item['icon'] }}"
+        />
+    @endif
+    <div class="flex flex-col gap-y-1">
+        @if ($item['heading'] ?? false)
+            <strong class="font-medium">
+                {{ $item['heading'] }}
+                @if ($key === 'orders')
+                    <graphql query="{customer{orders{items{number}}}}">
+                        <span
+                            slot-scope="{ data }"
+                            v-if="data"
+                        >
+                            (@{{ data.customer.orders.items.length ?? 0 }})
```

Comment:
can the <span be a <template with the v-if and slot-scope just like you did below, can you test if that works?